dgmt/build.gradle

```diff
@@ -8,14 +8,15 @@ repositories {
 //        //url "https://repo.spring.io/plugins-release/"
 //    }
     mavenLocal()
+    maven { url 'https://jitpack.io' }
 }
 
 android {
     compileSdkVersion 28
     buildToolsVersion "28.0.3"
 
     defaultConfig {
-        minSdkVersion 14
```

Comment:
This seems like a very large jump. Consider incrementing `minSdkVersion` a single version at a time.

hahahahahaha

Reply:
It's so authentic, it's like you've had PR reviews before.